### PR TITLE
add missing assertions of some test cases

### PR DESCRIPTION
**Pull request by SparkLee**
add missing assertions of some test cases

### CODE REVIEW

**Review by tsunghanjacktsai — CHANGES_REQUESTED** [review]
Hi @SparkLee ,

Don't forget to import the corresponding package for your changes.

**Comment by SparkLee** [comment]
> Hi @SparkLee ,
> 
> Don't forget to import the corresponding package for your changes.

It's done.

**Comment by SparkLee** [comment]
> > Hi @SparkLee ,
> > Don't forget to import the corresponding package for your changes.
> 
> It's done.

File: acl/src/test/java/org/apache/rocketmq/acl/common/AclSignerTest.java [hunk 1]
```diff
@@ -27,8 +28,9 @@ public void calSignatureExceptionTest(){
 
     @Test
     public void calSignatureTest(){
-        AclSigner.calSignature("RocketMQ","12345678");
-        AclSigner.calSignature("RocketMQ".getBytes(),"12345678");
+        String expectedSignature = "IUc8rrO/0gDch8CjObLQsW2rsiA=";
+        Assert.assertEquals(expectedSignature, AclSigner.calSignature("RocketMQ", "12345678"));
+        Assert.assertEquals(expectedSignature, AclSigner.calSignature("RocketMQ".getBytes(), "12345678"));
```
Comment (ShannonDing) [inline acl/src/test/java/org/apache/rocketmq/acl/common/AclSignerTest.java:33]: Assert.assertEquals(AclSigner.calSignature("RocketMQ", "12345678"), AclSigner.calSignature("RocketMQ".getBytes(), "12345678"));

**Pull request merged**